### ISSUE TITLE
Avoid build changes when no files actually changed

### ISSUE THREAD

**Issue by mgrsskls** [factorial-io/factorial-frontend-stack#27]
Whenever a build is triggered (changing a branch, saving a file), the js files in the build are changed, even though there might be no changes in the source files.

<img width="1440" alt="bildschirmfoto 2019-01-14 um 16 24 36" src="https://user-images.githubusercontent.com/1113765/51122273-c3383300-1819-11e9-90d5-d90ad6af51d8.png">

**Comment by mlnmln** [comment]
1. Build files should not be under version control. 😉 
2. I don't think this is enough for me to reproduce. 

Maybe share the repo (in private) with me and I could investigate.

**Comment by krisimmig** [comment]
@mlnmln @mgrsskls I don't think this is happening anymore, just tested with the current frontend stack setup.

**Comment by mlnmln** [comment]
@krisimmig 

No, still an open issue. 

Using `yarn start` 
* saved a CSS file 
* The JS build is triggered again
* the JS build contains a timestamp that changes with every build

![Screenshot 2019-06-27 at 10 08 24](https://user-images.githubusercontent.com/4355220/60248879-d4ef2d80-98c3-11e9-9f14-846d617621eb.png)
